crates/apps/src/lib/node/ledger/shell/prepare_proposal.rs

```diff
@@ -294,10 +293,10 @@ where
         super::replay_protection_checks(&tx, temp_state).map_err(|_| ())?;
 
         // Check fees and extract the gas limit of this transaction
+        // TODO: check if masp fee payment is required
```

Comment:
Can we start a practice of referencing a GitHub issue every time we write a TODO?